### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by hans-strudle**
I was playing with your project (which is amazing btw), and set up a Dockerfile to quickly build and spin up the server. Figured I would make a PR for it.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM golang:1.8
```
Comment (plutov) [inline Dockerfile]: Let's use 1.9 :)

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+WORKDIR /go/src/app
+
+COPY . .
+
+RUN go-wrapper download
```
Comment (plutov) [inline Dockerfile]: Why not just `install`?

File: README.md [hunk 3]
```diff
@@ -23,3 +23,9 @@ telnet localhost 8080
 ```
 go test ./... -bench=.
 ```
+
+### Docker
```
Comment (plutov) [inline README.md]: Let's move it to `Development`

**Comment by hans-strudle** [comment]
I bumped to version 1.9.

The `go-wrapper download` takes care of installing any 3rd party packages, sets up directories, etc. I removed it but had to mimic the directory setup to get it working, and now it won’t unnecessarily pull from Github.

If you add 3rd party dependencies later on this will stop working.

**Review by plutov — APPROVED** [review]

**Pull request merged**